### DISCUSSION TITLE
Bug reports should be in Issues

### DISCUSSION THREAD

**Discussion by Polda18** [requarks/wiki#7125] [General]
Honestly, the decision to use Discussions for bug reports kinda defeats the purpose of both Discussions and Issues. Discussions are honestly meant to be used for chitchat and kind of like open ended support for people having difficulties with setting up their wikis. For bug reports and feature requests, Issues is what is meant to be used on GitHub. I understand that you want to keep Issues clean. But that's not what the Issues section was created for. It was created for users to submit apparent bug reports or feature requests, and for project maintainers to respond to them. It isn't reserved solely just for project maintainers to keep track of chores.

Part of the reason why I dislike this decision is that Discussions isn't easy to keep track of on GitHub. Surely, I can pop in to see my created discussions or discussions I posted my reply in. But it requires me to chew through menus and it's not intuitive at the first glance. But Issues is accessible right from the dashboard with a single click. With a single click I can keep track of my bug reports and feature requests, and I can instantly see if the issue is being handled. Discussions don't have that.

I guess it's entirely up to you, but I strongly advice you to reconsider this decision. It makes it harder for us to report any bugs, since we are used to a system that's being used everywhere else, but for some reason here it is done differently.

And if you're lacking time to chew through many bug reports, why don't you hire some help? I haven't noticed any kind of "We're looking for volunteers to help us with project management" kind of thing anywhere here or anywhere on your websites. Your project powers tons of documentation sites, you have some responsibility to take care of. Namely countless of bug reports that are sometimes sitting literal months untouched. If you feel overwhelmed with the amount of reports piling up, why don't you ask for help? Not saying that me specifically I would be available (I am actually quite busy with my own projects as well), but I'm sure some volunteers would happily donate some of their time to help you with bug hunting.

**Comment by NGPixel** [comment]
We tried using Issues before and it was a total nightmare to manage. People either can't read or they post questions / help as bug reports. Are Discussions perfect? No, but it's definitely better than having to deal with Issues + getting hundreds of email notifications.

The goal is to have Discussions converted to Issues when they are deemed valid and actionable. This is not the case **yet** as getting the v3 beta out is the priority, so Issues are strictly for v3 at the moment.

Asking for volunteers is a good idea but requires time for vetting. This is something that can be looked into after the v3 beta launch.

**Reply by Polda18** [reply]
As I said, you do you. But it does generate inconvenience for us, normal users, who actually use and maintain the wiki, and need to report bugs we find. GitHub Issues is standard for bug reports. I guess it can be used differently, I don't think it's against GH ToS to use Discussions for bug reports instead of standard Issues. But it's not the intentional purpose. Hope that v3 beta gets out ASAP, if that's the priority. Some bugs here are actually severe, preventing the usage of the wiki or severily limiting it.
